admin/formatter/class-metabox-formatter.php

```diff
@@ -54,6 +54,13 @@ private function get_defaults() {
 			'metadesc_template' => '',
 			/* translators: After this sentence, the used keyword will be added (if present). This string is added to support screen readers. */
```

Comment:
I keep it this way because in our plugin 90% of the translator comments don't start with a capital. The WordPress docs give examples with lowercase. 

If there was a change for us I think it's better to do this in a new pull request.